### PR TITLE
New content from uncecomp

### PR DESCRIPTION
**Pull request by mbaudin47**
Added SIAM-UQ and Uncecomp articles and slides.

### CODE REVIEW

**Comment by mbaudin47** [comment]
This builds, but I could not check the generated PDFs: where are they ?

**Comment by mbaudin47** [comment]
@sofianehaddad I think that this is good for a review.

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +16,9 @@ script:
...
-  - cd uncecomp2017/Uncecomp2017-article-OT && pdflatex -halt-on-error uncecomp-2017-article-OpenTURNS.tex && pdflatex uncecomp-2017-article-OpenTURNS.tex && cd -
+  - cd uncecomp2017/Uncecomp2017-article-OT && pdflatex -halt-on-error uncecomp-2017-article-OpenTURNS.tex && bibtex uncecomp-2017-article-OpenTURNS && pdflatex uncecomp-2017-article-OpenTURNS.tex && pdflatex uncecomp-2017-article-OpenTURNS.tex && cd -
   - cd uncecomp2017/Uncecomp2017-slides-OT && pdflatex -halt-on-error uncecomp-2017-slides-OpenTURNS.tex && pdflatex uncecomp-2017-slides-OpenTURNS.tex && cd -
-  - cd uncecomp2019/Uncecomp2019-article-OT && pdflatex -halt-on-error uncecomp-2019-article-OpenTURNS.tex && pdflatex uncecomp-2019-article-OpenTURNS.tex && cd -
+  - cd uncecomp2019/Uncecomp2019-article-OT && pdflatex -halt-on-error uncecomp-2019-article-OpenTURNS.tex && bibtex uncecomp-2019-article-OpenTURNS && pdflatex uncecomp-2019-article-OpenTURNS.tex && pdflatex uncecomp-2019-article-OpenTURNS.tex && cd -
```
Comment (jschueller) [inline .travis.yml]: you need to add ```${TRAVIS_BUILD_DIR}/uncecomp*/*/*.pdf``` to line 32 of travis.yml for the generated pdf to be copied

**Comment by jschueller** [comment]
The pdfs are uploaded to github.io, but this happens only on master (ie after integration) and on a sucessful build

File: .travis.yml (hunk 1 above)
Comment (mbaudin47) [reply .travis.yml]: Ok : I'll fix this.

**Comment by mbaudin47** [comment]
@jschueller I think that this is good to integrate.

**Pull request merged**